src/objects.js

```diff
@@ -1,41 +1,64 @@
 const createPerson = (name, age) => {
-  // your code here
+  return { name, age };
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  if (object[property]) return true;
+  return false;
 };
 
 const isOver65 = person => {
-  // your code here
+  if (person.age > 65) return true;
+  return false;
 };
 
 const getAges = people => {
-  // your code here
+  const array = [];
+  people.forEach(element => {
+    array.push(element.age);
+  });
+  return array;
 };
 
-const findByName = (name, people) => {
-  // your code here
-};
+const findByName = (name, people) => people.find(obj => obj.name === name);
+{
+}
 
-const findHondas = cars => {
-  // your code here
-};
+const findHondas = cars => cars.filter(obj => obj.manufacturer === 'Honda');
+{
+}
 
 const averageAge = people => {
-  // your code here
+  const totalAge = people.reduce((previousAge, currentPerson) => {
+    return previousAge + currentPerson.age;
+  }, 0);
+  return totalAge / people.length;
 };
 
 const createTalkingPerson = (name, age) => {
-  // your code here
+  return {
+    name: name,
+    age: age,
+    introduce: strangersName => {
+      return (
```

Comment:
Nice, you could also use string interpolation here using back ticks ``Hi ${strangersName}, my name is ${name} and I am ${age}!``